modAionImpl/src/org/aion/zero/impl/sync/SyncStats.java

```diff
@@ -54,14 +54,7 @@
     private final Lock leechesLock = new ReentrantLock();
     private final boolean leechesEnabled;
 
-    /** @implNote Access to this resource is managed by the {@link #responsesLock}. */
-    private final Map<String, List<Long>> statusRequestTimeByPeers = new HashMap<>();
-    /** @implNote Access to this resource is managed by the {@link #responsesLock}. */
-    private final Map<String, List<Long>> statusResponseTimeByPeers = new HashMap<>();
```

Comment:
**Note:** In the previous implementation these two maps were expanded with new time recordings but these values were never deleted leading to memory issues.